knowledge_repo/app/static/js/pages/base.js

```diff
@@ -36,12 +36,24 @@ $(document).ready(function() {
     });
 
     $('#searchbar').keypress(function(event) {
-        var keycode = (event.keyCode ? event.keyCode : event.which);
-        if (keycode == '13') {
-            var path = document.location.pathname;
-            window.location = '/feed?filters=' + $('#searchbar').val()
+      var keycode = (event.keyCode ? event.keyCode : event.which);
+      if (keycode == '13') {
+          var path = document.location.pathname;
+          var search_path = window.location.search;
+          search_path = search_path.slice(1);
+          search_path_split = search_path.split('&');
+          url = '/feed?filters=' + $('#searchbar').val();
+          for(var i = 0;i < search_path_split.length;i ++)
+          {
```

Comment:
Can this not be made into a function?

Reply:
I mean, this piece of code is repeated twice in the code. It should typically be a function right?

Reply:
I will make it a function and update the pr

Reply:
is this done? @tanuj208